examples/mixed_language/tests/test_mixed_language.py

```diff
@@ -1,15 +1,25 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
 import cocotb
-from cocotb.triggers import Timer
+from cocotb.clock import Clock
+from cocotb.triggers import RisingEdge, Timer
 
 
 @cocotb.test()
-async def mixed_language_test(dut):
+async def mixed_language_accessing_test(dut):
     """Try accessing handles and setting values in a mixed language environment."""
     await Timer(100, units="ns")
 
     verilog = dut.i_swapper_sv
     dut._log.info("Got: %s" % repr(verilog._name))
 
+    # discover all attributes of the SV component
+    # This is a workaround since SV modules are not discovered automatically
+    # when using Modelsim/Questa and a VHLD toplevel file.
```

Comment:
```suggestion
    # when using Modelsim/Questa and a VHDL toplevel file.
```
Sorry, can't help it :)